alchemiscale/storage/statestore.py

```diff
@@ -96,15 +97,29 @@ def _select_task_from_taskpool(taskpool: Subgraph) -> Union[ScopedKey, None]:
     return chosen_one[0]
 
 
+def _select_tasks_from_taskpool(taskpool: List[Tuple[str, float]], count) -> List[str]:
```

Comment:
Since we have this, can we delete `_select_task_from_taskpool` above?

Reply:
Removed